cpp/open3d/ml/contrib/RoIPoolKernel.cu

```diff
@@ -0,0 +1,300 @@
+/*
+Point cloud feature pooling
+Written by Shaoshuai Shi
+All Rights Reserved 2018.
+*/
+
+#include <math.h>
+#include <stdio.h>
+
+#include "open3d/ml/contrib/RoIPoolKernel.h"
+
+#define THREADS_PER_BLOCK 256
+#define DIVUP(m, n) ((m) / (n) + ((m) % (n) > 0))
+// #define DEBUG
+
+__device__ inline int pt_in_box3d(float x,
```

Comment:
Similar to other `contrib` ops, could you put the code inside

```
namespace open3d {
namespace ml {
namespace contrib {
```

Same for other files.